### PR TITLE
Added xkcd feature to fun module

### PR DESCRIPTION
**Pull request by Darkuuu**

### CODE REVIEW

File: src/util/embed.py
Comment (CookieDecide) [inline src/util/embed.py:1]: We need to consider the consistency in our file naming.
util_audio_player.py already exists in the main branch, so we ned to agree upon one style here.

File: src/util/xkcd.py
Comment (CookieDecide) [inline src/util/xkcd.py:1]: Same as on embed.py

File: src/util/embed.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
+"""Copyright by CookieDecide, Darkuuu
+Licensed under MIT License
+Contains various helper functions to create embeds for sending. 
+Assures consistent look over different modules.
+"""
+
+import discord
+from discord.ext import commands
+
+
```
Comment (CookieDecide) [inline src/util/embed.py:10]: Again concerning the consistency, in util_audio_player.py we used a seperate class as a wrapper for the implemented methods, so i would recommend to the same style for all.

File: src/util/xkcd.py [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    """
+    max_number = 0
+    try:
+        # Get current comic to derive maximum amount of available comics
+        with urllib.request.urlopen(_url_comic_current) as url:
+            data = json.load(url)
+            max_number = data["num"]
+            rand_number = random.randint(1, max_number)
+            return create_xkcdcomic(rand_number)
+    except Exception as err:
```
Comment (CookieDecide) [inline src/util/xkcd.py:49]: It is not good practice to catch a generic exception, try to use specific exceptions that can occure here or at leat log the exception.

File: src/util/xkcd.py [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+    Returns:
+        XkcdComic: Valid comic object.
+        None: Only returned when not able to fetch comic.
+    """
+    url_comic = _url_comic_left + str(number) + _url_comic_right
+    try:
+        with urllib.request.urlopen(url_comic) as url:
+            data = json.load(url)
+            return XkcdComic(data["title"], data["img"], number)
+    except Exception as err:
```
Comment (CookieDecide) [inline src/util/xkcd.py:66]: Same as above comment.

File: src/util/embed.py [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+        title: Title of the embed.
+        description: Description/Message of embed.
+        img_url: URL pointing to the image.
+        url: URL pointing to the webpage the image is embedded on.
+             If not existing, provide img_url here.
+
+    Returns:
+        An embed with blue color showing an image.
+    """
+    color = 0x14D8FA
```
Comment (CookieDecide) [inline src/util/embed.py:24]: Maybe we can agree to add a comment with the approximate color, like "blue", for better readability.

**Review by CookieDecide — CHANGES_REQUESTED** [review]
Take a look at the comments and maybe get in contact to agree on some of the mentioned issues in consistency.

File: src/util/embed.py
Comment (Darkuuu) [reply src/util/embed.py:1]: Good point, will call you later today

File: src/util/xkcd.py
Comment (Darkuuu) [reply src/util/xkcd.py:1]: See above

File: src/util/embed.py (hunk 1 above)
Comment (Darkuuu) [reply src/util/embed.py:10]: See above

File: src/util/xkcd.py (hunk 2 above)
Comment (Darkuuu) [reply src/util/xkcd.py]: Fixed, catching now HTTP error and log it if something goes wrong here. Anything else should be considered a bug if happening.

File: src/util/xkcd.py (hunk 3 above)
Comment (Darkuuu) [reply src/util/xkcd.py]: See above

File: src/util/embed.py (hunk 4 above)
Comment (Darkuuu) [reply src/util/embed.py:24]: Again, will call you later regarding design decisions in code. Maybe adding a seperate module to specify different colour schemes might be an idea

**Comment by Darkuuu** [comment]
Please take a look already at the exception fix, regading code design decisions I will call you later today

**Review by CookieDecide — APPROVED** [review]

**Pull request merged**